presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -343,18 +343,18 @@ public boolean isFinished()
         return finished;
     }
 
+    // This operator can receive input from the upstream operator only when all of following conditions are true:
```

Comment:
`// This operator needs input ...`

let's keep the comment consistent with the function name